### ISSUE TITLE
[BUG] Search for 'vm' in hypervisor capabilities is broken

### ISSUE THREAD

**Issue by davidjumani** [apache/cloudstack-primate#523]
**Describe the bug**
When a user searches for 'vm' in hypervisor capabilities, [on the first time the page loads] it breaks (endlessly loads) and errors out in the console

![Screenshot from 2020-07-09 11-13-56](https://user-images.githubusercontent.com/8244774/87001533-54048a80-c1d5-11ea-9bd5-f419d6935e25.png)

**Comment by davidjumani** [comment]
Probably related to the backend since VMware has duplicate keys

```
(qa) 🐱 > list hypervisorcapabilities keyword=vmware filter=hypervisor,id
    {
      "hypervisor": "VMware",
      "id": "11"
    },
    {
      "hypervisor": "VMware",
      "id": "12"
    },
    {
      "hypervisor": "VMware",
      "id": "0b7ad266-5fe0-11ea-9a56-1e006800018c"
    },
    {
      "hypervisor": "VMware",
      "id": "15eda318-5fe0-11ea-9a56-1e006800018c"
    }
```

**Comment by davidjumani** [comment]
Extra logs :

![Screenshot from 2020-07-09 11-52-07](https://user-images.githubusercontent.com/8244774/87004188-a85e3900-c1da-11ea-84bf-d1add1234bb8.png)

**Comment by Pearl1594** [comment]
Firstly, an odd observation - I noticed that the search only breaks when done on page 1 
Was able to however fix the issue by ensuring uniqueness in the rowkeys in ListView

**Comment by rhtyd** [comment]
@davidjumani is this still an issue or fixed?

**Comment by davidjumani** [comment]
Fixed in #564
